### PR TITLE
feat: warp route screen pan on key combo; fix its tooltip

### PR DESCRIPTION
**Pull request by EttyKitty**
- Disabled automatic warp route screen pan, as it was just obnoxious. Very often it was just sending your screen flying when you didn't need that.
- Changed manual pan trigger from just clicking to SHIFT+click, to prevent missclicks.
- Fixed tooltip flipping not working on the star map.

### CODE REVIEW

**Review by OH296 — CHANGES_REQUESTED** [review]
minor issue is there is now no way to travel to the end point of a lane while a fleet is selected as clicking with the left will destroy the obj fleet select which makes long distance  lanes essentially unusable as who can be assed to manually scroll across the map and find  a name

**Comment by EttyKitty** [comment]
> as clicking with the left will destroy the obj fleet select

Well, that's not good. Let's see if I can do something.

**Comment by EttyKitty** [comment]
@OH296, is it viable to just disable the fleet deselection on click if shift is held?

**Comment by OH296** [comment]
> @OH296, is it viable to just disable the fleet deselection on click if shift is held?

i think so, i know the code so i can sort a pr to here tomorrow

**Comment by EttyKitty** [comment]
> i think so, i know the code so i can sort a pr to here tomorrow

Thanks. I'll leave this to you, as I can't quickly find where it happens.

**Comment by OH296** [comment]
Is the delay on merging this down to the tooltip disappearing? minor enough issue imo but if so i can look into it a bit later?

**Comment by EttyKitty** [comment]
> Is the delay on merging this down to the tooltip disappearing? minor enough issue imo but if so i can look into it a bit later?

Well, tbh I would like to solve it before merge. It's a pretty minor error but very nasty imo.
But if it's a headache - I can merge this and we think of a fix later.

**Comment by OH296** [comment]
> > Is the delay on merging this down to the tooltip disappearing? minor enough issue imo but if so i can look into it a bit later?
> 
> Well, tbh I would like to solve it before merge. It's a pretty minor error but very nasty imo. But if it's a headache - I can merge this and we think of a fix later.

i don't know if it's tricky or not as yet but i have a nasty feeling it will be time consuming. leave it a few days but if it's not done soonish just merge would be my current position

**Comment by EttyKitty** [comment]
Could it be that the tooltip fix was this easy? Was there a reason why you disallow drawing the tooltip with shift held?

**Pull request merged**
